src/legacy/core_plugins/dashboard_embeddable_container/public/np_ready/public/lib/embeddable/grid/dashboard_grid.test.tsx

```diff
@@ -17,50 +17,56 @@
  * under the License.
  */
 
-import '../../np_core.test.mocks';
-
-import React from 'react';
-import { shallowWithIntl, nextTick, mountWithIntl } from 'test_utils/enzyme_helpers';
 // @ts-ignore
 import sizeMe from 'react-sizeme';
 
+import React from 'react';
+import { shallowWithIntl, nextTick, mountWithIntl } from 'test_utils/enzyme_helpers';
 import { skip } from 'rxjs/operators';
-
-import { EmbeddableFactory } from '../../../../embeddable_api/public';
-import {
-  ContactCardEmbeddableFactory,
-  CONTACT_CARD_EMBEDDABLE,
-} from '../../../../embeddable_api/public/test_samples';
-
+import { EmbeddableFactory, GetEmbeddableFactory } from '../../embeddable_api';
 import { DashboardGrid, DashboardGridProps } from './dashboard_grid';
 import { DashboardContainer } from '../dashboard_container';
 import { getSampleDashboardInput } from '../../test_helpers';
+import {
+  CONTACT_CARD_EMBEDDABLE,
+  ContactCardEmbeddableFactory,
+} from '../../../../../../../embeddable_api/public/np_ready/public/lib/test_samples/embeddables/contact_card/contact_card_embeddable_factory';
 
 let dashboardContainer: DashboardContainer | undefined;
 
 function getProps(props?: Partial<DashboardGridProps>): DashboardGridProps {
-  const embeddableFactories = new Map<string, EmbeddableFactory>();
-  embeddableFactories.set(CONTACT_CARD_EMBEDDABLE, new ContactCardEmbeddableFactory());
+  const __embeddableFactories = new Map<string, EmbeddableFactory>();
+  __embeddableFactories.set(
+    CONTACT_CARD_EMBEDDABLE,
+    new ContactCardEmbeddableFactory({} as any, (() => {}) as any, {} as any)
+  );
+  const getEmbeddableFactory: GetEmbeddableFactory = (id: string) => __embeddableFactories.get(id);
+
   dashboardContainer = new DashboardContainer(
     getSampleDashboardInput({
       panels: {
         '1': {
           gridData: { x: 0, y: 0, w: 6, h: 6, i: '1' },
           type: CONTACT_CARD_EMBEDDABLE,
-          explicitInput: { firstName: 'Bob', id: '1' },
+          explicitInput: { id: '1' },
         },
         '2': {
           gridData: { x: 6, y: 6, w: 6, h: 6, i: '2' },
           type: CONTACT_CARD_EMBEDDABLE,
-          explicitInput: { firstName: 'Stacey', id: '2' },
```

Comment:
Why remove?  I see it's not related to the tests so it could be removed, but I am just wondering if it's throwing type errors and that's why you removed it, or if you just felt it was unnecessary so removed.

Reply:
Because of TypeScript type checking:

![image](https://user-images.githubusercontent.com/9773803/62041431-f19cbf00-b1fb-11e9-8609-443af7a2a293.png)